### ISSUE TITLE
Windows qinglong branch – Multiple install/runtime issues on fresh setup (Python version conflict + SyntaxError on server start)

### ISSUE THREAD

**Issue by hazencruz** [sdbds/ACE-Step-1.5-for-windows#22]
> **📚 Before submitting:** Have you tried [DeepWiki AI Assistant](https://deepwiki.com/ace-step/ACE-Step-1.5)?
no but i tried grok which is how i found a solution

### Describe the bug

Two main issues on fresh install of the qinglong Windows fork:

Dependency resolution failure during install and model download. Keeps trying to use Python 3.11, but flash-attn and nano-vllm have no compatible wheels for 3.11 on Windows x86_64.

SyntaxError when starting the backend: duplicate argument 'skip_existing' in function definition in preprocess_to_tensors.

**These, of course, prevent the server from starting cleanly, causing 500: fetch failed / ECONNREFUSED errors in the UI.**

### To Reproduce

Delete old folder and run fresh clone:
git clone --recurse-submodules https://github.com/sdbds/ACE-Step-1.5-for-windows.git -b qinglong
Run the install script: .\1、install-uv-qinglong.ps1
Run server: .\3、run_server.ps1
Seeing repeated dependency errors about Python 3.11 / flash-attn during install and model download.

_Initial setup says "models downloaded" but then they download again when running the server. Consider changing message._

### Expected behavior

Install script should default to (or easily allow) Python 3.12 on Windows.
Server should start without SyntaxError after model download.
Backend should be reachable so the UI can generate songs without 500 fetch failed errors.

User should not see "models downloaded successfully" until models are downloaded successfully.

OS: Windows 11
GPU: RTX 4090 24GB
Browser: Chrome (latest)
ACE-Step Version: qinglong branch (cloned today)

### I was able to work around the issues with these steps:

Force Python 3.12 using uv python install 3.12 + uv venv --python 3.12 --seed
Pre-install Torch: uv pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu128
Comment out flash-attn in pyproject.toml to let install complete
Manually edit acestep\training\dataset_builder_modules\preprocess.py and remove the duplicate skip_existing=False from the preprocess_to_tensors function definition.

After these changes the models download successfully and the server starts (though flash-attn is disabled). Generation works once backend is running.
With RTX 4090 performance is still very good even without flash-attn.

Would be awesome if the install script enforced Python 3.12 by default and the skip_existing argument conflict was cleaned up in the dataset builder files (unless getting this more up to date with the current main branch of ACE 1.5 would just resolve that).

**Comment by sdbds** [comment]
Thank you for letting me know. The version fix issue has now been resolved. Previously, the fixed version was AMD64, which prevented x86 64 from installing the Windows version 3.11 of flashattn.
